src/Stratis.Bitcoin.Tests/Utilities/DateTimeProviderTest.cs

```diff
@@ -10,9 +10,11 @@ public class DateTimeProviderTest
         [Fact]
         public void GetUtcNowReturnsCurrentUtcDateTime()
         {
+            int timeOffset = -96;
```

Comment:
I think you mixed up your PRs here.

Reply:
Yes, I will quickly move the change to the other PR